boms/cloud-oss-bom/pom.xml

```diff
@@ -69,10 +69,13 @@
     <dependencies>
       <dependency>
         <groupId>com.google.guava</groupId>
-        <artifactId>guava-bom</artifactId>
+        <artifactId>guava</artifactId>
+        <version>${guava.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.google.guava</groupId>
+        <artifactId>guava-testlib</artifactId>
```

Comment:
We really shouldn't include this one. guava-gwt is intended for clients, though we don't have to include it if we aren't already. guava-testlib isn't.